src/components/ExhibitionCard/index.jsx

```diff
@@ -13,9 +13,9 @@ const ExhibitionCard = ({ id, title, shortDescription, imageId, imageUrl, status
 
   return (
     <div className={styles.container}>
-      <Link to="/exhibitions" key={id}>
+      <Link to={`/exhibitions/${id}`} key={id}>
```

Comment:
You can remove the key prop from this component, it's not needed since you're not rendering a list of elements.